src/ngraph_builder.cc

```diff
@@ -458,15 +458,17 @@ static Status TranslateArgMaxOp(
   TF_RETURN_IF_ERROR(GetStaticInputVector(op, 1, static_input_map, &tf_dim));
 
   ng::Shape input_shape = ng_input->get_shape();
-  size_t input_rank = ng_input->get_shape().size();
+  size_t input_rank = input_shape.size();
+
+  tf_dim.size() == 1;
 
```

Comment:
This generates a boolean, but does not assert.
We need to return tf error status if the size != 1

Reply:
👍